spiketoolkit/sorters/sorterlist.py

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Is this file just keeping track of all sorters that are installed?

Reply:
It is a list for all sorter and all installed sorter.

So for testing sorters on a particular dataset the list will generate automatiocally with future helper fuction.

Reply:
I see. Sounds good.